python/cuxfilter/charts/core/aggregate/core_aggregate.py

```diff
@@ -130,14 +122,11 @@ def compute_stride(self):
             self.stride_type = float
 
         if self.stride is None and self.data_points is not None:
-            if self.stride_type == int:
-                self.stride = self.stride_type(
-                    round((self.max_value - self.min_value) / self.data_points)
-                )
-            else:
-                self.stride = self.stride_type(
-                    (self.max_value - self.min_value) / self.data_points
-                )
+            raw_stride = (self.max_value - self.min_value) / self.data_points
+            stride = (
+                round(raw_stride) if self.stride_type == int else raw_stride
+            )
```

Comment:
We have such a short mandated line length (no action suggested, just :cry:)

Reply:
I know :(